dev-docs/bidders/adagio.md

```diff
@@ -21,6 +21,26 @@ sidebarType: 1
 
 The Adagio bidder adapter requires setup and approval from the Adagio team. Please reach out to [contact@adagio.io](mailto:contact@adagio.io) for more information.
 
+### Configuration
+
+#### User Sync
+
+Adagio strongly recommends enabling user syncing through iFrames. This functionality improves DSP user match rates and increases the bid rate and bid price. Make sure to call pbjs.setConfig() only once. This configuration is optional in Prebid, but required by Adagio
```

Comment:
pbjs.setConfig() must be displayed as code plz

Add a `.` at the end of the phrase.

Reply:
Fixed